.circleci/config.yml

```diff
@@ -51,9 +57,9 @@ jobs:
 
       - run: api/typescript/scripts/build.sh
 
-  publish_typescript:
+  publish_api_typescript:
     docker:
-      - image: cimg/node:14.17.1
+      - image: "cimg/node:<< pipeline.parameters.node-version>>"
     steps:
       - checkout
       - run: api/typescript/scripts/build.sh
```

Comment:
I _think_ the `package-path` variable below will need to be `api/typescript` (since we moved it a level deeper now)

Reply:
fixed